packages/react-components/react-combobox/src/stories/Dropdown/DropdownBestPractices.md

```diff
@@ -5,9 +5,12 @@
 
 ### Do
 
+- **Provide a label for the Dropdown.**
+- **Set the Option's `value` prop if the content contains JSX.** The Option value is used for keyboard accessibility to enable users to type a letter or string and jump to the matching option. The value is calculated from the children by default, but if the Option contains JSX, the `value` prop should be used to directly provide a string value.
```

Comment:
Value (or label) is going to be required, so this will need to be reworded.

Reply:
I think I'll make the update in the other PR, so there's no mismatch between the doc and the behavior in the meantime